packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/explorerKey.from.event..st

```diff
@@ -0,0 +1,9 @@
+key handling
+explorerKey: aChar from: view event: event
+
+	event anyModifierKeyPressed ifFalse: [^ false].
+
+	self currentSelection ifNotNil: [
```

Comment:
Ich persönlich finde es einfacher, den Sinn zu erkennen, wenn die Bedingung tatsächlich als Konjunktion und nicht verschachtelt dargestellt wird.

```smalltalk
(self currentSelection notNil) & (aChar == $b)
	ifTrue: [self browseMethodCall].
```

Oder gleich mit early exit:

```smalltalk
self currentSelection ifNil: [^ false].
aChar == $b ifTrue: [self browseMethodCall].
```

Reply:
The current way of implementing this is easier to extend if we were to add new keybindings in the future.

Reply:
Ich finde besonders die folgende Variante die Till vorgeschlagen hat gut lesbar: 

```smaltalk 
(self currentSelection notNil) & (aChar == $b)
	ifTrue: [self browseMethodCall].
```
Hier ist ganz klar der Sinn zu erkennen.